--- a/src/framework/Makefile
+++ b/src/framework/Makefile
@@ -21,7 +21,7 @@ SHFLAGS += -Wl,-soname,$(SONAME)
 endif
 
 ifeq ($(targetos), Linux)
-SHFLAGS += -Wl,--version-script=mlt.vers
+SHFLAGS += 
 endif
 
 OBJS = mlt_frame.o \
--- a/src/mlt++/Makefile
+++ b/src/mlt++/Makefile
@@ -19,11 +19,11 @@ endif
 ifneq (, $(shell $(CXX) --version | grep -is -e g++ -e clang))
 	CXXFLAGS += -std=c++11
 endif
-CXXFLAGS += -I.. $(RDYNAMIC) -DVERSION=\"$(version)\" -fvisibility=hidden
+CXXFLAGS += -I.. $(RDYNAMIC) -DVERSION=\"$(version)\"
 
 LDFLAGS := -L../framework -lmlt $(LDFLAGS)
 ifeq ($(targetos), Linux)
-LDFLAGS += -Wl,--version-script=mlt++.vers
+LDFLAGS += 
 endif
 
 OBJS = MltAnimation.o \
