--- mlt-0.9.0/src/framework/Makefile~	2014-05-27 15:11:26.000000000 +0400
+++ mlt-0.9.0/src/framework/Makefile	2014-05-27 15:12:45.600000000 +0400
@@ -21,7 +21,7 @@ SHFLAGS += -Wl,-soname,$(SONAME)
 endif
 
 ifeq ($(targetos), Linux)
-SHFLAGS += -Wl,--version-script=mlt.vers
+SHFLAGS += 
 endif
 
 OBJS = mlt_frame.o \
--- mlt-0.9.0/src/mlt++/Makefile~	2014-05-27 15:40:13.304000000 +0400
+++ mlt-0.9.0/src/mlt++/Makefile	2014-05-27 15:40:30.836000000 +0400
@@ -19,11 +19,11 @@ SONAME = $(NAME).$(soversion)
 LIBFLAGS += -Wl,-soname,$(SONAME)
 endif
 
-CXXFLAGS += -I.. $(RDYNAMIC) -DVERSION=\"$(version)\" -fvisibility=hidden
+CXXFLAGS += -I.. $(RDYNAMIC) -DVERSION=\"$(version)\"
 
 LDFLAGS += -L../framework -lmlt
 ifeq ($(targetos), Linux)
-LDFLAGS += -Wl,--version-script=mlt++.vers
+LDFLAGS += 
 endif
 
 OBJS = MltConsumer.o \
