--- mlt-0.9.2/configure~	2014-10-17 15:40:03.000000000 +0400
+++ mlt-0.9.2/configure	2014-10-17 15:41:38.368000000 +0400
@@ -292,63 +292,6 @@ export moduledir=${libdir}/mlt-${soversi
 export mltdatadir=${datadir}/mlt-${soversion}
 fi
 
-# Double check MMX (Darwin, Linux and FreeBSD supported, may end up disabling MMX on other platforms incorrectly)
-if [ "$mmx" = "true" ]
-then
-	case $targetos in
-		Darwin)
-		sysctl -a hw | grep "mmx: 1" > /dev/null || mmx=false
-		;;
-		Linux)
-		grep mmx /proc/cpuinfo > /dev/null 2>&1 || mmx=false
-		;;
-		FreeBSD)
-		[ "$(make -V MACHINE_CPU:Mmmx -f /dev/null)" ] || mmx=false
-		;;
-		*)
-		grep mmx /proc/cpuinfo > /dev/null 2>&1 || mmx=false
-		;;
-	esac
-fi
-
-# Double check SSE (Darwin, Linux and FreeBSD supported, may end up disabling SSE on other platforms incorrectly)
-if [ "$sse" = "true" ]
-then
-	case $targetos in
-		Darwin)
-		sysctl -a hw | grep "sse: 1" > /dev/null || sse=false
-		;;
-		Linux)
-		grep sse /proc/cpuinfo > /dev/null 2>&1 || sse=false
-		;;
-		FreeBSD)
-		[ "$(make -V MACHINE_CPU:Msse -f /dev/null)" ] || sse=false
-		;;
-		*)
-		grep sse /proc/cpuinfo > /dev/null 2>&1 || sse=false
-		;;
-	esac
-fi
-
-# Double check SSE2 (Darwin, Linux and FreeBSD supported, may end up disabling SSE2 on other platforms incorrectly)
-if [ "$sse2" = "true" ]
-then
-	case $targetos in
-		Darwin)
-		sysctl -a hw | grep "sse2: 1" > /dev/null || sse2=false
-		;;
-		Linux)
-		grep sse2 /proc/cpuinfo > /dev/null 2>&1 || sse2=false
-		;;
-		FreeBSD)
-		[ "$(make -V MACHINE_CPU:Msse2 -f /dev/null)" ] || sse2=false
-		;;
-		*)
-		grep sse2 /proc/cpuinfo > /dev/null 2>&1 || sse2=false
-		;;
-	esac
-fi
-
 # Show help if requested
 if [ $help = 1 ]
 then
